docs/image-preview.md

```diff
@@ -188,3 +188,19 @@ If the image shows properly when using Überzug++ independently, but not when us
 - The contents of your log file.
 - The contents of `/tmp/ueberzugpp-$USER.log`.
 - A GIF demonstration of the above steps.
+
+## What Image formats are supported?
+
+Supported formats may depend on the protocol used.
```

Comment:
Thanks for the PR, but this isn't correct. 

Yazi supports image formats regardless of protocol restrictions. If a format isn't supported by the protocol itself, Yazi will convert it to comply with the protocol requirements.

Reply:
@sxyazi Thanks. I'll update this 👍